src/js/actions/index.js

```diff
@@ -0,0 +1,206 @@
+                        return dogWithImg;
+                    }
+                    return dog;
+                });
+                dispatch(setDogImg(dogWithImg, newDogList));
+            })
+            .catch(error => {
+                const dogWithImgError = { ...dog, imgError: error.toString() };
+                const newDogList = dogList.map(dog => {
+                    if (dog.name === dogBreed) {
+                        return dogWithImgError;
+                    }
+                    return dog;
+                });
+                dispatch(setDogImg(dogWithImgError, newDogList));
+            });
+    };
+}
+
+// Calls action dispatch to set dogList on storage
+// Params:
+//  dogList (array) = an array of objects containing the dogList
+export function setDogList(dogList) {
+    // sets the new list in local storage
+    localStorage.setItem('dogList', JSON.stringify(dogList));
+    // returns the action to set the list in store
+    return {
+        type: types.SET_DOG_LIST,
+        dogList
+    };
+}
+
+// If there is an error retrieving the dogList call action to set error on store
+// Params:
+//  error (string) = the error that ocurred
+function setDogListError(dogListError) {
+    return {
+        type: types.SET_DOG_LIST_ERROR,
+        dogListError
+    };
+}
+
+// Calls action dispatch to set dog image on storage
+// Params:
+//  dog (obj) = object of the dog to replace in storage
+export function setDogImg(dog, dogList) {
+    // sets the new list in local storage
+    localStorage.setItem('dogList', JSON.stringify(dogList));
+    // returns the action to set the list in store
+    return {
+        type: types.SET_DOG_IMG,
+        dog
+    };
+}
+
+// Calls action dispatch to set dog image error on storage
+// Params:
+//  dog (obj) = object of the dog to replace in storage
+export function setDogImgError(dog, dogList) {
+    // sets the new list in local storage
+    localStorage.setItem('dogList', JSON.stringify(dogList));
+    // returns the action to set the list in store
+    return {
+        type: types.SET_DOG_IMG_ERROR,
+        dog
+    };
+}
+
+// Call action to set the submitted dog in the store
+// Params:
+//  submittedDog (formDate) = formData Object of the submitted dog
+function setSubmittedDog(submittedDog) {
+    return {
+        type: types.SET_SUBMITTED_DOG,
+        submittedDog
+    };
+}
+
+// Handles messages list in store
+// Params:
+//  message (object) = object with message attributes: message(string), type(string)
+export function setMessage(msg) {
+    const message = { ...msg, id: Math.floor(Math.random() * Math.floor(999999999)) };
+    return {
+        type: types.SET_MESSAGE,
+        message
+    };
+}
+
+// Removes a message from message list in storage
+// Params:
+//  error (string) = the error that ocurred
+export function removeMessage(message_id) {
+    return {
+        type: types.REMOVE_MESSAGE,
+        message_id
```

Comment:
```suggestion
        messageId
```
Don't mix up snake_case and camelCase

Reply:
bump